src/xb-builder-source.c

```diff
@@ -471,6 +474,7 @@ xb_builder_source_get_istream (XbBuilderSource *self,
 			return NULL;
 		xb_builder_source_remove_last_extension (basename);
 		g_set_object (&priv->istream, istream_tmp);
+		file = NULL;
```

Comment:
this makes sense, but can you add a comment explaining how it's only valid for the "outer" one please -- then LGTM.

Reply:
Good suggestion, done, thanks